compiler/one-cmds/one-create-quant-dataset

```diff
@@ -0,0 +1,143 @@
+''''export SCRIPT_PATH="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)" # '''
+''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
+''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
+''''echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command." # '''
+''''exit 255                                                                            # '''
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import h5py as h5
+import numpy as np
+import argparse
+import glob
+import os
+import onelib.utils as oneutils
+
+
+def get_parser():
+    """Create and return given the argument parser"""
+    parser = argparse.ArgumentParser(
+        description='command time tool to convert data files to hdf5 file')
+    parser.add_argument("-d",
+                        "--data_type",
+                        type=str,
+                        help="Data type of either rawdata2hdf5 or numpy2hdf5",
+                        required=True)
+    parser.add_argument(
+        "-l",
+        "--data_list",
+        type=str,
+        help=
+        "Path to the text file where lists the absolute paths of data files to be converted.",
+        required=True)
+    parser.add_argument("-o",
+                        "--output_path",
+                        type=str,
+                        help="Path to the output hdf5 file.",
+                        required=True)
+    return parser
+
+
+def verify_args(parser, args):
+    """Verify the given arguments"""
+
+    def is_valid_attr(args, attr):
+        return hasattr(args, attr) and getattr(args, attr)
+
+    # check if requried arguments is given
+    missing = []
+    if not is_valid_attr(args, 'data_type'):
+        missing.append('-d/--data_type')
+    if not is_valid_attr(args, 'data_list'):
+        missing.append('-l/--data_list')
+    if not is_valid_attr(args, 'output_path'):
+        missing.append('-o/--output_path')
+    if len(missing):
+        parser.error('the following argument are requried: ' + ' '.join(missing))
+
+    # check if data_type is correct
+    if getattr(args, 'data_type') != "rawdata2hdf5" and getattr(
+            args, 'data_type') != "numpy2hdf5":
+        parser.error(
+            'the following argument needs to be modified to rawdata2hdf5 or numpy2hdf5: -d/--data_type'
+        )
```

Comment:
Ditto. Maybe you can use "[choice](https://docs.python.org/ko/3/library/argparse.html#choices)" feature.

Reply:
Thank you a lot :)